static/app/components/badge/groupPriority.tsx

```diff
@@ -98,8 +99,14 @@ export function makeGroupPriorityDropdownOptions({
 }
 
 export function GroupPriorityBadge({priority, children}: GroupPriorityBadgeProps) {
+  const organization = useOrganization();
   return (
-    <StyledTag type={getTagTypeForPriority(priority)}>
+    <StyledTag
+      type={getTagTypeForPriority(priority)}
+      isBigtag={organization.features.includes(
```

Comment:
Was this added to make this match the size of the assignee dropdown? We could probably just update this to be the larger size regardless of the assignee dropdown being release or not

Reply:
That was the intent, yes. Definitely a very small detail, but I just wanted to limit the scope of changes in this PR to be completely under the feature flag - plus I feel like it would be a little weird if users saw a fatter priority button with no other UI changes